test/fixtures/config.json

```diff
@@ -38,5 +38,26 @@
       "S7",
       "S8"
     ]
+  },
+  "devices": {
```

Comment:
Yeah, I think this needs to happen. I see a few classes of potential devices:

* IR (lirc)
* Zwave
* HTTP based (IoT devices with an API)
* RF (lirc, using `irsend SIMULATE`)
* etc